src/pybind/mgr/dashboard/run-frontend-unittests.sh

```diff
@@ -27,18 +27,12 @@ fi
 npm run lint --silent
 if [ $? -gt 0 ]; then
   failed=true
-  echo -e "\nTry running 'npm run lint -- --fix' to fix some linting errors."
-fi
-
-npm run prettier:lint --silent
-if [ $? -gt 0 ]; then
-  failed=true
-  echo -e "\nTry running 'npm run prettier' to fix linting errors."
+  echo -e "\nTry running 'npm run fix' to fix some linting errors."
```

Comment:
Each linter already outputs the list of errors and I only know if they exist after running it, so prepending `Linting error` is not an option.

I could change the message to  this:
`Try running 'npm run fix' to fix some linting errors. Some errors might need a manual fix.`
Would that resolve this issue?

Reply:
Hm, I may wasn't communicating clear enough. I want to improve the message which is shown on checks when a PR is opened (Jenkins). The linter does output the list of errors if run manually and normally, but this code runs it using `--silent` which I expect not to show any output. 

I might be wrong, but when I tried to find the problem of the check on my first PR I wasn't able to see that an error occurred just by looking at the logs, because the only thing I saw was the **suggestion to run something** to fix linting errors, but not the clear messsage that **an error occurred that needs to be fixed**.  

Did I miss to see something like that and does the log output say that a linting error occurred that needs to be fixed? If so, ignore what I said. If not, I really think we should improve that message here, either in this PR or another.

PS: I like the improved version of the echo'd text.

Reply:
The `--silent` flag is only for npm outputs, so we don't have the following:
![image](https://user-images.githubusercontent.com/399326/46076369-40656c80-c185-11e8-8d6f-054947f865ca.png)

Instead we will get:
![image](https://user-images.githubusercontent.com/399326/46076403-5b37e100-c185-11e8-818e-1d2d3930f54d.png)

All 3 linter will output the list of errors, so there should always be some information about them.

I have just pushed a commit with lint errors so we can make sure it is showing in the jenkins.

Reply:
I vote for using the --silent flag.